Dockerfile

```diff
@@ -1,20 +1,16 @@
 FROM quay.io/ukhomeofficedigital/docker-centos-base
 
-RUN mkdir -p /workdir/node /workdir/sassc /workdir/libsass /app
+RUN mkdir -p /opt/node /app
 
-WORKDIR /workdir/node
+WORKDIR /opt/node
 RUN yum install -y curl && \
-    yum -y groupinstall "Development Tools" && \
-    curl http://nodejs.org/dist/node-latest.tar.gz | tar xz --strip-components=1 && \
-    ./configure && \
-    make install && \
-    yum clean all && \
-    rm -rf /workdir/node
-
-RUN rm -rf /workdir
+    curl https://nodejs.org/dist/latest/node-v4.0.0-linux-x64.tar.gz | tar xz --strip-components=1 && \
+    ln -s /opt/node/bin/node /usr/local/bin/node && \
+    ln -s /opt/node/bin/npm /usr/local/bin/npm
```

Comment:
I'd just do something like that:

```
RUN mkdir -p /nodejs && curl -s https://nodejs.org/dist/v0.12.7/node-v0.12.7-linux-x64.tar.gz | tar -xzf - -C /nodejs --strip-components=1

ENV PATH=${PATH}:/nodejs/bin
```

Instead of symlinking particular binaries, because you cannot be sure what other binaries nodejs distribution might start shipping.